src/cascadia/WindowsTerminal/IslandWindow.h

```diff
@@ -6,6 +6,8 @@
 #include <winrt/TerminalApp.h>
 #include "../../cascadia/inc/cppwinrt_utils.h"
 
+#define CM_NOTIFY_FROM_TRAY (WM_APP + 1)

```

Comment:
Do be careful with this one! BaseWindow specifies its own window message as well... so we need to make sure we do not collide!

Reply:
Put them both in their own little file since I'll need to include it in the `TrayIcon.cpp/.h` coming in the generic tray icon PR